Smarty/templates/Inventory/ProductDetails.tpl

```diff
@@ -146,14 +146,25 @@ function displayCoords(currObj,obj,mode,curr_row)
 		<table width="100%"  border="0" cellspacing="0" cellpadding="1">
 			<tr>
 			<td class="small" valign="top">
-				<input type="text" id="productName1" name="productName1" class="small" style="width:70%" value="{if isset($PRODUCT_NAME)}{$PRODUCT_NAME}{/if}" readonly />
+				<div class="slds-combobox_container slds-has-inline-listbox cbds-product-search">
+					<div class="slds-combobox slds-dropdown-trigger slds-dropdown-trigger_click slds-combobox-lookup" aria-expanded="false" aria-haspopup="listbox" role="combobox">
+						<div class="slds-combobox__form-element slds-input-has-icon slds-input-has-icon_right" role="none">
+							<input id="productName1" class="slds-input slds-combobox__input cbds-inventoryline__input--name" aria-autocomplete="list" aria-controls="listbox-unique-id" autocomplete="off" role="textbox" placeholder="{$APP.typetosearch_prodser}" value="{if isset($PRODUCT_NAME)}{$PRODUCT_NAME}{/if}" type="text" style="box-shadow: none;">
+							<span class="slds-icon_container slds-icon-utility-search slds-input__icon slds-input__icon_right">
+							<svg class="slds-icon slds-icon slds-icon_x-small slds-icon-text-default" aria-hidden="true">
+								<use xlink:href="lib/LDS/icons/utility-sprite/svg/symbols.svg#search" xmlns:xlink="http://www.w3.org/1999/xlink"></use>
+							</svg>
+							</span>
+						</div>
+					</div>
+				</div>
 				<input type="hidden" id="hdnProductId1" name="hdnProductId1" value="{if isset($PRODUCT_ID)}{$PRODUCT_ID}{/if}" />
 				{if $PRODUCT_OR_SERVICE eq 'Services'}
 					<input type="hidden" id="lineItemType1" name="lineItemType1" value="Services" />
-					&nbsp;<img id="searchIcon1" title="{'Services'|@getTranslatedString:'Services'}" src="{'services.gif'|@vtiger_imageurl:$THEME}" style="cursor: pointer;" align="absmiddle" onclick="servicePickList(this,'{$MODULE}',1)" />
+					{*&nbsp;<img id="searchIcon1" title="{'Services'|@getTranslatedString:'Services'}" src="{'services.gif'|@vtiger_imageurl:$THEME}" style="cursor: pointer;" align="absmiddle" onclick="servicePickList(this,'{$MODULE}',1)" />*}
```

Comment:
I would not eliminate these buttons as the popup search is more powerful than the autocomplete and you can even create the product/service